targets/signing_config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mediaType": "application/vnd.dev.sigstore.signingconfig.v0.1+json",
+  "caUrl": "https://fulcio.sigstore.dev",
+  "oidcUrl": "https://oauth2.sigstore.dev/auth/",
```

Comment:
```suggestion
  "oidcUrl": "https://oauth2.sigstore.dev/auth",
```

Reply:
I noticed this in staging too. I’m not sure if this would cause an error, but for uniformity, can we drop the slash?

Reply:
Fixed.